proprietary/tokens/src/common/amsterdam/spacing.tokens.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think our spacing system will be a bit more complicated than this, for now we can just hardcode the values in the component level design tokens imo

Reply:
Actually I hope not. I my understanding spacings values should be used to compose other values. Preferable no values for every implementation. This can be extended of course, but I guess this should be the main reference.